src/fab/index.tsx

```diff
@@ -23,15 +23,8 @@ export interface FabProps extends RMWC.WithRippleProps {
 /** A floating action button component */
 export const Fab = withRipple({ surface: false })(
   createComponent<FabProps>(function Fab(props, ref) {
-    const {
-      children,
-      label,
-      icon,
-      trailingIcon,
-      mini,
-      exited,
-      ...rest
-    } = props;
+    const { children, label, icon, trailingIcon, mini, exited, ...rest } =
```

Comment:
Do you have prettier installed? I think if not we'll keep incurring formatting only changes. I'd have to check but we might be missing a prettier config file in the repo.

Reply:
I do have prettier and it should be enabled.. hmm